### DISCUSSION TITLE
The Goal: Continuity

### DISCUSSION THREAD

**Discussion by aaronjensen** [aaronjensen/software-development#11] [Articles]
https://github.com/aaronjensen/software-development/blob/master/continuity.md

**Comment by alexbunardzic** [comment]
Great article. You have nicely described the ever creeping accidental complexity, which is the public enemy number one in software development/delivery.

**Reply by aaronjensen** [reply]
Thank you.

**Comment by Porkstone** [comment]
Enjoyed your article. Maintaining productivity as you scale is a complex problem with so many contributory factors. Just adding one more person to a team will make managing communication much more challenging. Creating strong shared beliefs/norms is the key to tackling the exponential increase in lines of communication that comes with adding people to the team.

This diagram shows how increasing a team from six to seven people increases the lines of communication from 15 to 21
![image](https://github.com/user-attachments/assets/e3d03538-fe6f-4067-af59-4a81aafa76cd)

**Reply by aaronjensen** [reply]
Thank you. Yes, this is true, and it's one reason we spend so much time together as an entire team. We have a daily sync (not standup) where we can talk about anything that we need to talk about. We share new capabilities, norms, and standards; talk about mistakes we encountered in our code; raise design concerns; etc. Anyone can put a topic on the board and we don't remove that topic until it is truly resolved.

Sync can last anywhere from 15 minutes to 2 hours, depending on what we need to talk about. We also keep work logs and people are expected to at least scan each others'. 

Oh, and many of us wear multiple hats, so the titles in the diagrams you posted would be anathema.
